index.js

```diff
@@ -2,56 +2,28 @@
 // Node version, if it isn't it will throw the following error to inform
 // you.
 if (Number(process.version.slice(1).split(".")[0]) < 16) throw new Error("Node 16.x or higher is required. Update Node on your system.");
-
-try {
-  require("./config.js");
-} catch (e) {
-  console.log("Creating a config.js file for the bot");
-  require("./setup.js");
-}
+require("dotenv").config();
 
 // Load up the discord.js library
 const { Client, Collection} = require("discord.js");
 // We also load the rest of the things we need in this file:
 const { readdirSync } = require("fs");
-const Enmap = require("enmap");
-const config = require("./config.js");
-
+const { intents, partials, permLevels } = require("./config.js");
+const logger = require("./modules/Logger.js");
 // This is your client. Some people call it `bot`, some people call it `self`,
 // some might call it `cootchie`. Either way, when you see `client.something`,
 // or `bot.something`, this is what we're referring to. Your client.
 const client = new Client({
-  intents: config.intents,
-  partials: config.partials
+  intents: intents,
+  partials: partials
```

Comment:
couldn't this line be `intents, partials` instead of 
```js
  intents: intents,
  partials: partials
```

Reply:
Ooh, very good point! I'll fix em